test/algos/adaptors/test_let_stopped.cpp

```diff
@@ -98,7 +98,7 @@ TEST_CASE("let_stopped function is not called on regular flow", "[adaptors][let_
 }
 TEST_CASE("let_stopped function is not called on error flow", "[adaptors][let_stopped]") {
   bool called{false};
-  error_scheduler<int> sched;
+  error_scheduler<int> sched{};
```

Comment:
This is good, but I think I would prefer to change `error_scheduler` to initialize its data member on default initialization.

Reply:
Thanks. Done, with an in-class default member initializer.